### ISSUE TITLE
[IntelliJ][ReportedByUser][2020.1 EAP] When create a new project, show this Uncaught Exception  java.lang.Throwable: Cannot add an action twice: Publish Group (Publish Group)

### ISSUE THREAD

**Issue by v-lanjli** [microsoft/azure-tools-for-java#4107]
**IntelliJ build version:** 2020.1 EAP IC-201.6073.9
**OS:** Windows 10
**JDK:** JetBrains s.r.o 11.0.6
**Plugin version:** https://dev.azure.com/mseng/VSJava/_build/results?buildId=11679560&view=artifacts&type=publishedArtifacts
**Additional Info:** None

**Repor Step:**
Create a new project in an already opened project

**Parent component:** 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,8,1601x860,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
**Error message:** 
```
IdeaLoggingEvent[message=null, throwable=java.lang.Throwable: Cannot add an action twice: Publish Group (Publish Group)
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:146)
	at com.microsoft.intellij.AzurePlugin.initializeAzureActionGroup(AzurePlugin.java:164)
	at com.microsoft.intellij.AzurePlugin.initComponent(AzurePlugin.java:148)
```

**Comment by t-rufang** [comment]
Fixed in #4148

**Comment by v-lanjli** [comment]
Verified and fixed in this build (https://dev.azure.com/mseng/VSJava/_build/results?buildId=11760285&view=artifacts&type=publishedArtifacts)
Closed this issue.
